docs/en/query_language/dicts/external_dicts_dict_sources.md

```diff
@@ -84,6 +84,16 @@ Example of settings:
     <http>
         <url>http://[::1]/os.tsv</url>
```

Comment:
Why we cannot include credentials in URL:
http://user:password@[::1]/os.tsv

Reply:
I suspect a bit the code who handle authentication to not handle it if we set the auth on the `URI`, I tried to do some test locally with an HTTP server, the server returned a `401` error if we put the auth on the URL.

https://github.com/ClickHouse/ClickHouse/blob/master/dbms/src/IO/ReadWriteBufferFromHTTP.h#L112-L113